packages/bpk-component-slider/src/BpkSlider.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import Slider from 'react-slider';
+import { cssModules } from 'bpk-react-utils';
+import React from 'react';
+
+import STYLES from './bpk-slider.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkSlider = (props) => {
+  const {
+    min,
+    max,
+    value,
+    step,
+    large,
+    minDistance,
+    className,
+    onChange,
+    onAfterChange,
+  } = props;
+
+  const sliderClass = [getClassName('bpk-slider')];
+  const handleClass = [getClassName('bpk-slider-handle')];
```

Comment:
the handle and bar elements seem to be children of `bpk-slider` - this means that they should have classnames of `bpk-slider__handle` & `bpk-slider__bar` respectively